src/features/defi/components/Overview/Overview.tsx

```diff
@@ -107,17 +109,20 @@ export const Overview: React.FC<OverviewProps> = ({
         <Stack spacing={0}>
           <Stack p={8} spacing={6}>
             <Flex flexDir='column' gap={3}>
-              {onAccountIdChange && (
+              {onAccountIdChange && accountId && (
                 <>
-                  <Flex gap={4} alignItems='center' width='full' justifyContent='space-between'>
+                  <InlineCopyButton
+                    isDisabled={isUtxoAccountId(accountId)}
```

Comment:
q: are we going to add this one as well for THORFi (i.e savers here)?   If so, parents should probably pass an `positionAddress` prop of sorts

Reply:
I'm not sure to get this one, for now UTXOs are disabled, but all other opportunities contains this copy button, they actually don't need this prop for now

But, in the future, if we wan't to add copy buttons to all those places for UTXOs, it might be the trick yes